GPR183.py

```diff
@@ -0,0 +1,184 @@
+# Importing modules
+import os
+from Bio import SeqIO
+from Bio.SeqUtils import gc_fraction
+from typing import Union
```

Comment:
Не совсем корректно представлены импорты. Корректирую все программы с помощью isort)

```suggestion
import os
from typing import Union

from Bio import SeqIO
from Bio.SeqUtils import gc_fraction
```

Reply:
Насколько я знаю, так необходимо делать, если импортируются какие-то кастомные (собственные) пакеты. В данном случае, пакет Bio таким не является и мне кажется, не обязательно их разделять пустой строкой.

Reply:
1) Встроенные
2) Сторонние
3) Локальные

Между ними по строке, так что Катя (в лице isort'a) и isort (в лице Кати) правы